tests/accuracy/test_measure_optics.py

```diff
@@ -12,6 +12,7 @@
 from .twiss_to_lin import optics_measurement_test_files
 
 LIMITS = {'P': 1e-4, 'B': 3e-3, 'D': 1e-2, 'A': 6e-3}
+LIMITS_3BPM = {'P': 1e-4, 'B': 3e-3, 'D': 1e-2, 'A': 0.95}
```

Comment:
The limit for alpha function is not far off from situation, where LHC's arc are consistent with zero. I know the discrepancy comes mostly from IR, however pushes the limit on the test in this form too high. We should probably rethink the test of 3-BPM method ... what about test on PS lattice?

Reply:
3 BPM tests removed, will be implemented at a later point, likely as part of #88